tests/cypress/support/e2e.ts

```diff
@@ -35,7 +35,7 @@ declare global {
       checkApplicationStatus(appName: string, clusterName?: string): Chainable<Element>;
       deleteApplicationDeployment(clusterName?: string): Chainable<Element>;
       modifyDeployedApplication(appName: string, clusterName?: string): Chainable<Element>;
-      createRoleTemplate(roleType: string, roleName: string, newUserDefault?: string['yes'|'no'], verbs?: string[], resources?: string[], apiGroups?: string[], nonResourcesURLs?: string[] ): Chainable<Element>;
+      createRoleTemplate(roleType: string, roleName: string, newUserDefault?: string['yes'|'no'], verbs?: string[], rules?: string[], apiGroups?: string[], nonResourcesURLs?: string[] ): Chainable<Element>;
```

Comment:
Here not sure whether it should contain `verbs` which is part of `rules`.

```suggestion
      createRoleTemplate(roleType: string, roleName: string, newUserDefault?: string['yes'|'no'], rules?: string[], apiGroups?: string[], nonResourcesURLs?: string[] ): Chainable<Element>;
```